package.json

```diff
@@ -50,7 +50,7 @@
     "@rollup/plugin-json": "^4.1.0",
     "@rollup/plugin-node-resolve": "^13.1.3",
```

Comment:
All errors are fixed except for: https://github.com/ExtraHorizon/javascript-sdk/pull/848/files and the one where we need to a major upgrade on axios.